yvalenta/search.rb

```diff
@@ -0,0 +1,57 @@
+class SearchVowels
+
+  def initialize
+    @count = 0
+    vowels
+  end
+
+  def open_file
+    f = File.open("../text.txt", "r")
+    run(f)
+    f.close
+  end
+
+  def time_start(start)
+    struct = Struct.new(:start)
```

Comment:
For burning face,
but i would like to know in which cases can be applied struct